LeagueInformer/LeagueInformer/Program.cs

```diff
@@ -14,46 +14,55 @@ public class Program
 
         public static void Main(string[] args)
         {
-            if (ConnectionService.HasInternetConnection())
+            while (true)
             {
-                Console.WriteLine(AppResources.Main_WelcomeUser);
-                Console.WriteLine(AppResources.Main_ChooseFunction);
-                Console.WriteLine("1. Opcja nr 1");
-                Console.WriteLine(AppResources.MainMenu_GetChallengerList);
-                Console.WriteLine(AppResources.MainManu_AboutApp);
-                Console.WriteLine(AppResources.Main_Quit);
-                var option = Console.ReadLine();
-                while (option != null && (!option.Equals("1") || !option.Equals("2") || !option.Equals("3") || !option.Equals("4")))
+                if (ConnectionService.HasInternetConnection())
                 {
-                    switch (option)
+                    MainMenu();
+                    var option = Console.ReadLine();
+                    while (option != null && (!option.Equals("1") || !option.Equals("2") || !option.Equals("3") || !option.Equals("4")))
                     {
-                        case "1":
-                            FirstOption();
-                            break;
-                        case "2":
-                            GetBestChallengers().Wait();
-                            break;
-                        case "3":
-                            AboutApp();
-                            break;
-                        case "4":
-                            Environment.Exit(1);
-                            break;
-                        default:
-                            Console.WriteLine(AppResources.Common_OptionIsNotAvailable);
-                            option = Console.ReadLine();
-                            break;
+                        switch (option)
+                        {
+                            case "1":
+                                FirstOption();
+                                break;
+                            case "2":
+                                GetBestChallengers().Wait();
+                                break;
+                            case "3":
+                                AboutApp();
+                                break;
+                            case "4":
+                                Environment.Exit(1);
+                                break;
+                            default:
+                                Console.WriteLine(AppResources.Common_OptionIsNotAvailable);
+                                option = Console.ReadLine();
```

Comment:
Jeżeli już mamy pobieranie opcji w pętli (while(true)) to po co dwa razy pobierać opcję jeżeli jest spoza zakresu?